elasticdl/python/common/save_utils.py

```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
```

Comment:
It seems that in the next PR, we should remove the filename from `save_utils.py` into `checkpoint.py`.

Reply:
Exactly, `checkpoint.py` is more meaningful than `save_utils.py`